### ISSUE TITLE
Vocalization of shortcuts

### ISSUE THREAD

**Issue by cyberbaloo** [numerique-gouv/meet#875]
## Bug Report

**Problematic behavior**
When you know the visio shortcuts (control+D / control+E) and use them, they are not announced by screen readers.

**Expected behavior/code**
When the shortcuts are activated, there should be an alert saying that the camera and microphone are turned off/on.

**Steps to Reproduce**
1. Open a new meeting
2. Activate the control+D or control+E shortcuts
3. And then the bug happens!

**Comment by cyberbaloo** [comment]
Same issue for these buttons even before entering the meeting. The camera and microphone status are not announced.
